code/modules/mob/living/carbon/human/ai/brain/ai_brain_helpers.dm

```diff
@@ -141,11 +141,12 @@
 
 /datum/human_ai_brain/proc/unholster_any_weapon()
```

Comment:
switching to help intent means they're gonna get stuck by shuffling into each other a lot

Reply:
True, but also they are trying to execute things when in disarm or PB when in harm. At this moment we might unironically switch to grab intent for melee combat